### PR TITLE
feat: Add Client Request Time to Context

### LINKED ISSUE #2199: 🚀 [Feature] Add Timestamp to Request Header in All Clients
## 🚀 Feature
Add a timestamp of now, including the timezone, in the request header of both clients (wallet and admin interface).
The purpose is to have this timestamp in the context object for the resolver.

### PR DESCRIPTION
**Pull request by Mogge**
## 🍰 Pullrequest
Client sends client request time in request header to server. Server reads client request time and adds it to resolver context as string

### Issues
- fixes #2199 

### Todo

- [X] None

### CODE REVIEW

File: backend/src/server/context.ts [hunk 1]
```diff
@@ -18,13 +19,16 @@ export interface Context {
 
 const context = (args: ExpressContext): Context => {
   const authorization = args.req.headers.authorization
-  let token: string | null = null
+  const clientRequestTime = args.req.headers.clientrequesttime
```
Comment (clauspeterhuebner) [inline backend/src/server/context.ts:22]: clientrequesttime now all in lowercase is correct?

**Review by clauspeterhuebner — APPROVED** [review]

**Review by clauspeterhuebner — CHANGES_REQUESTED** [review]
I merge this solution to #2225 and the context will only contain the clientRequestTime as soon as a token will be in the context too.
So it seems your solution will only add the clientRequestTime for token authenticated requests.

File: backend/src/server/context.ts (hunk 1 above)
Comment (Mogge) [reply backend/src/server/context.ts:22]: Yes, it looks like the request headers are all lowercase. I tried it first in camel case

**Comment by Mogge** [comment]
You are right, the time is only available if authenticated. I think this is enough for now, as you need authentication to do any time related operations.

**Review by clauspeterhuebner — APPROVED** [review]

**Pull request merged**